### ISSUE TITLE
Is there a way to build a .css (for production)?

### ISSUE THREAD

**Issue by peterbarraud** [cody-greene/scssify#32]
Maybe use some syntax like: 
```
browserify MyComponent.js -t [scssify --output bundle.css] >bundle.js
```

**Comment by cody-greene** [comment]
No. If you want to compile a standalone `.css` file just use node-sass directly.
```
$ npm install node-sass
$ node_modules/.bin/node-sass \
    ./src/styles/index.scss \
    >build/bundle.css \
    --output-style compressed
```

**Comment by peterbarraud** [comment]
I have implemented a onSendToCSSFile functionality that sends the CSS output to the client.

See the project I've created at: [Browserify with Budo & SCSSIFY](https://github.com/peterbarraud/rapo-browserify-budo-scssify)

The reason I've taken this approach, is that `scssify` is pretty neat at handling `SCSS` require statements in the `index.js`. In my project, if you install `bulma` and uncomment the require statement in `index.js`, scssify works perfect in the livereload. But if I put this functionality in, I can use it also to [build](https://github.com/peterbarraud/rapo-browserify-budo-scssify/blob/master/scripts/build.js).

So I've created the onSendToCSSFile  callback in the [`scssify/lib/index.js`](https://github.com/peterbarraud/rapo-browserify-budo-scssify/blob/master/scssify/index.js)
